karma/flows/test_controllers.coffee

```diff
@@ -765,3 +765,51 @@ describe 'Controllers:', ->
 
         for rule_test in rule_tests
           expect(scope.isRuleComplete(rule_test['rule'])).toBe(rule_test['complete'])
+
+          
+     it 'should generate json quick replies to send', ->
+      loadFavoritesFlow()
+
+      actionset = flowService.flow.action_sets[0]
+      action = actionset.actions[0]
+    
+      json_quick_responses = [{
+                    'title':'Quick reply',
+                    'payload':'Test quick reply is ok',
+                    'content_type':'text'
+                }]
+
+      editAction actionset, action, (scope) ->
+        scope.AddNewQuickResponse()
```

Comment:
This shouldn't be capitalized. Why not pass the initialization to this function instead of setting them below?